### DISCUSSION TITLE
Error Failed to download package 'Microsoft.Net.Native.Compiler.2.0.3' from 'https://api.nuget.org/v3-flatcontainer/microsoft.net.native.compiler/2.0.3/microsoft.net.native.compiler.2.0.3.nupkg'. The HTTP request to 'GET https://api.nuget.org/v3-flatcontainer/microsoft.net.native.compiler/2.0.3/microsoft.net.native.compiler.2.0.3.nupkg' has timed out after 100000ms.

### DISCUSSION THREAD

**Discussion by kenishashank** [NuGet/Home#11559] [Q&A]
In my Visual studio 2019, there is a library error and I tried on multiple places for a solution but failed. Please Suggest I solve this. These are errors showing-

Error Failed to download package 'runtime.win10-arm64.Microsoft.Net.Native.Compiler.2.2.3' from '[https://api.nuget.org/v3-flatcontainer/runtime.win10-arm64.microsoft.net.native.compiler/2.2.3/runtime.win10-arm64.microsoft.net.native.compiler.2.2.3.nupkg'](https://api.nuget.org/v3-flatcontainer/runtime.win10-arm64.microsoft.net.native.compiler/2.2.3/runtime.win10-arm64.microsoft.net.native.compiler.2.2.3.nupkg%27). The HTTP request to 'GET https://api.nuget.org/v3-flatcontainer/runtime.win10-arm64.microsoft.net.native.compiler/2.2.3/runtime.win10-arm64.microsoft.net.native.compiler.2.2.3.nupkg' has timed out after 100000ms.

Error Failed to download package 'Microsoft.Net.Native.Compiler.2.0.3' from '[https://api.nuget.org/v3-flatcontainer/microsoft.net.native.compiler/2.0.3/microsoft.net.native.compiler.2.0.3.nupkg'](https://api.nuget.org/v3-flatcontainer/microsoft.net.native.compiler/2.0.3/microsoft.net.native.compiler.2.0.3.nupkg%27). The HTTP request to 'GET https://api.nuget.org/v3-flatcontainer/microsoft.net.native.compiler/2.0.3/microsoft.net.native.compiler.2.0.3.nupkg' has timed out after 100000ms.

Please help me to solve this. Thank you for your concentration.

**Comment by zivkan** [comment]
Here are nuget.org team's docs on how to investigate and report connection/performance problems: https://docs.microsoft.com/en-us/nuget/nuget-org/nuget-org-faq#why-can-t-i-download-packages-from-or-upload-packages-to-nuget-org
